### ISSUE TITLE
[Bug] use chia plotnft show, the height of wallet 2 does not change

### ISSUE THREAD

**Issue by tangdasan** [Chia-Network/chia-blockchain#11015]
### What happened?

I have multiple full nodes (with the same key). Yesterday, I upgraded a host to 1.3.2. 

When I use chia plotnft show, the height of wallet 2 does not change, which causes my full node to connect to the pool abnormally, and there are no points!

 Showing my old pool because the height is not up to date.

### Version

1.3.2

### What platform are you using?

Linux

### What ui mode are you using?

CLI

### Relevant log output

```shell
Wallet height: 1785851
Sync status: Synced
Wallet id 2: 
Current state: FARMING_TO_POOL
Current state from block height: 569217
```

**Comment by wallentx** [comment]
Does the height update after you restart the services? I'm not able to replicate this on my farm currently.

**Comment by tangdasan** [comment]
> Does the height update after you restart the services? I'm not able to replicate this on my farm currently. How many blocks behind is `chia plotnft show` from `chia wallet show`?

I think I solved it, copied the pool address of version 1.2.11
